### ISSUE TITLE
Allow right click paste

### ISSUE THREAD

**Issue by alltheseas** [damus-io/notedeck#507]

**Comment by kuba-04** [comment]
Draft as of know, you might have a better idea, let me know pls

**Comment by alltheseas** [comment]
@kuba-04 have you reviewed [#contributing.md](https://github.com/damus-io/damus/blob/22f2aba96979c0f36d2b7165db08635db43774f4/docs/CONTRIBUTING.md?plain=1#L4)

**Comment by kuba-04** [comment]
you are right, sorry for not being careful with this. I just edited the commit name, but maybe you are also referring to some other issues?
